src/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import ReactDOM from "react-dom"
+import {Provider} from "react-redux"
+import {createStore, applyMiddleware} from "redux"
+import thunk from "redux-thunk"
+import rootReducer from "./reducer"
+import * as action_creators from "./action_creators"
```

Comment:
I don't like that I have to pass actionCreators down. Feels like something the components shouldn't know about until they use them directly.